### DISCUSSION TITLE
如何自动打开excel文件 (Translated: How to automatically open excel files)

### DISCUSSION THREAD

**Discussion by adaiana** [exceljs/exceljs#1906] [Q&A]
有什么代码可以自动打开对应路径下的excel文件吗

_Translated: Is there any code to automatically open the excel file in the corresponding path?_

**Comment by Siemienik** [comment]
I/O operations:

https://github.com/exceljs/exceljs/blob/master/README_zh.md#%E6%96%87%E4%BB%B6-io

For the future. Please write in english :)
